apps/journeys/src/components/blocks/Icon/Icon.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactElement } from 'react'
+import {
+  CheckCircle,
+  PlayArrow,
+  Translate,
+  RadioButtonUnchecked,
+  FormatQuote,
+  LockOpen,
+  ArrowForward,
+  ChatBubbleOutline,
+  LiveTv,
+  MenuBook
+} from '@mui/icons-material'
+import { ButtonFields_startIcon as IconType } from '../../../../__generated__/ButtonFields'
+
+export interface IconProps extends IconType {
+  ariaHidden?: boolean
+}
+
+export function Icon({ name, color, size }: IconProps): ReactElement | null {
+  const fontSize =
+    size === 'sm'
+      ? '16px'
+      : size === 'md'
+      ? '20px'
+      : size === 'lg'
+      ? '28px'
+      : size === 'xl'
+      ? '48px'
+      : 'inherit'
+
+  switch (name) {
+    case 'CheckCircle':
+      return (
+        <CheckCircle color={color ?? undefined} sx={{ fontSize: fontSize }} />
```

Comment:
You can shorthand the property below
```suggestion
        <CheckCircle color={color ?? undefined} sx={{ fontSize }} />
```